From 178ff56eb809fc7d5bf0ef99e49fec407ff167fe Mon Sep 17 00:00:00 2001
From: novenary <novenary@kwak.zip>
Date: Mon, 27 Jan 2025 16:44:52 +0200
Subject: [PATCH 1/3] Build offline

---
 tools/build/build.js | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/tools/build/build.js b/tools/build/build.js
index 40e0b9bb..4ef522ed 100755
--- a/tools/build/build.js
+++ b/tools/build/build.js
@@ -126,18 +126,22 @@ export const BuildModlistTarget = new Juke.Target({
   inputs: ['manifest.json'],
   outputs: ['dist/modlist.html'],
   executes: async ({ get }) => {
+    /*
     if (!env.CFCORE_API_TOKEN && !get(KeyParameter)) {
       Juke.logger.error('CFCORE_API_TOKEN env var is required for downloading mods.');
       throw new Juke.ExitCode(1);
     }
+    */
     fs.mkdirSync("dist", { recursive: true })
     const jsonData = JSON.parse(fs.readFileSync('manifest.json', 'utf-8'));
     let html = '<ul>\n'
+    /*
     for (const key in jsonData.files) {
       const file = jsonData.files[key];
       const modInfo = await GetModInfo(env.CFCORE_API_TOKEN ?? get(KeyParameter), file.projectID);
       html += `<li><a href=${modInfo.links.websiteUrl}>${modInfo.name} (by ${modInfo.authors[0].name})</a></li>\n`;
     }
+    */
     html += '</ul>'
     fs.writeFileSync('dist/modlist.html', html);
   }
@@ -147,6 +151,7 @@ export const DownloadModsTarget = new Juke.Target({
   inputs: ['manifest.json'],
   outputs: () => [], // always run, we have internal logic to check mods now
   executes: async ({ get }) => {
+    return;
     if (!env.CFCORE_API_TOKEN && !get(KeyParameter)) {
       Juke.logger.error('CFCORE_API_TOKEN env var is required for downloading mods.');
       throw new Juke.ExitCode(1);
-- 
2.47.1

