From c3a8bfb6ab67728dfef9b1b3db2dac23cc15b865 Mon Sep 17 00:00:00 2001
From: Dusk Banks <me@bb010g.com>
Date: Fri, 21 Mar 2025 02:19:58 -0700
Subject: [PATCH 3/5] Enable NBT tooltips by default

---
 config/cucumber-client.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/cucumber-client.toml b/config/cucumber-client.toml
index ee9484a1..5564a851 100644
--- a/config/cucumber-client.toml
+++ b/config/cucumber-client.toml
@@ -4,5 +4,5 @@
 	#Enable tag list tooltips for blocks/items?
 	tagTooltips = true
 	#Enable NBT tooltips for items?
-	nbtTooltips = false
+	nbtTooltips = true
 
-- 
2.47.2

