From 67c368b234943cc0bc9cf36125a49b105cee3df9 Mon Sep 17 00:00:00 2001
From: novenary <novenary@kwak.zip>
Date: Mon, 27 Jan 2025 19:02:09 +0200
Subject: [PATCH 3/3] Build fat client zip

---
 tools/build/build.js | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/tools/build/build.js b/tools/build/build.js
index 4ef522ed..afed5ef7 100755
--- a/tools/build/build.js
+++ b/tools/build/build.js
@@ -251,6 +251,13 @@ export const BuildClientTarget = new Juke.Target({
       fs.cpSync(folders, `dist/client/${folders}`, { recursive: true })
     }
 
+    cpSyncFiltered('dist/modcache/', 'dist/client/mods', file => {
+      const fillet = file.toLowerCase();
+      return (
+        fillet.includes('.jar')
+      )
+    })
+
     await packMod("client");
   }
 })
-- 
2.47.1

